### PR TITLE
chore: create issue templates

### PR DESCRIPTION
**Pull request by latonz**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
```
Comment (CommonGuy) [inline .github/ISSUE_TEMPLATE/bug_report.md:31]: Is this section really necessary?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
```
Comment (CommonGuy) [inline .github/ISSUE_TEMPLATE/bug_report.md:28]: Do we need the browser information? Wouldn't the .NET version and Mapperly version make more sense?

**Review by CommonGuy — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (latonz) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Nope, I just reused the default template of GH. Adjusted to our project.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (latonz) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Nope, I just reused the default template of GH. Adjusted to our project.

**Review by CommonGuy — APPROVED** [review]

**Pull request merged**